src/containers/execution-environment-detail/base.tsx

```diff
@@ -207,7 +207,19 @@ export function withContainerRepo(WrappedComponent) {
                   promise
                     .then((results) => {
                       const task = results.find((x) => x.data && x.data.task);
-                      this.setState({ editing: false, loading: true });
+                      this.setState({
+                        editing: false,
+                        loading: true,
+                        alerts: alerts.concat({
+                          variant: 'success',
+                          title: (
+                            <Trans>
+                              Saved changes to execution environment &quot;$
```

Comment:
here's the extra `$` :)
```suggestion
                              Saved changes to execution environment &quot;
```